### DISCUSSION TITLE
TYPES [@types/pdfkit]

### DISCUSSION THREAD

**Discussion by ManuelRengifo19** [DefinitelyTyped/DefinitelyTyped#67187] [Issues with a @types package]
I am having an issue using pdfkit when I am building this library, this error is described below:

Error: node_modules/@types/pdfkit/index.d.ts:603:23 - error TS1110: Type expected.
603     type PDFSubsets = `PDF/A-1${"" | "a" | "b"}` | `PDF/A-2${"" | "a" | "b"}` | `PDF/A-3${"" | "a" | "b"}`; 

I am using Angular V 11 and node npm@7.15.0 and 14.17.3.
If there is anybody who sometimes has the same issue and knows how could I solve it, this information would be useful.

[a comment by an automated account is not shown]

**Reply by BamButz** [reply]
I will investigate this issue during the day.

**Reply by ymendozaCity** [reply]
were you able to solve this issue?

**Reply by BamButz** [reply]
I have to admit that this got lost on my list. I tried to replicate it just yet, and it seems that this is most likely induced by an old version of typescript.
